newspaper/extractors/image_extractor.py

```diff
@@ -58,7 +58,7 @@ def _get_favicon(self, doc: lxml.html.Element) -> str:
         )
         if meta:
             favicon = parsers.get_attribute(meta[0], "href")
-            return favicon
+            return favicon if favicon is not None else ""
```

Comment:
I think this works too:
`return favicon or ""`

Reply:
It does. I tend towards more verbose options and not liking relying on truthy values v.s. explicit tests, but I've changed it to the form you suggested.